index.html

```diff
@@ -45,7 +45,7 @@ <h1 class="primary-text">I'm Oscar <br> Glad to see you!</h1>
           you see and have a project you need coded, don't hestiate to contact
```

Comment:
- [ ] The flash animations you added to the headline title is nice 👎 but it would be nice if there is a way for the user to pause, stop, or hide the flashes because flashing or blinking content can cause seizures in people with photosensitive epilepsy. Kindly read more about it [here](https://www.w3.org/WAI/test-evaluate/preliminary/#moving)